docs/source/index.rst

```diff
@@ -11,14 +11,26 @@ with Numba, we suggest you start with the :doc:`User manual <user/index>`.
 
 
 .. toctree::
-   :numbered:
+   :caption: For all users
    :maxdepth: 2
 
    user/index.rst
    reference/index.rst
+
+
+.. toctree::
+   :caption: For GPU users
+   :maxdepth: 2
+
    cuda/index.rst
    cuda-reference/index.rst
    roc/index.rst
+
+
+.. toctree::
+   :caption: For advanced users and developers
```

Comment:
Locally, this ends up too wide for the sidebar, I can just see `For advanced users and developer` (and half of the `s`), any ideas?

Reply:
I struggled with this one (hadn't noticed the `s` was missing). `For advanced users / developers`, `For advanced users & developers`?
Alternative I could remove the `For` in all captions.

Reply:
Either of / or & look good to me. I think keeping the "For" would be nice.

Reply:
Agree with @gmarkall